### PR TITLE
add nearest_interp_v2 trt plugin

### PR DESCRIPTION
**Pull request by wangxinxin08**
### PR types
New features

### PR changes
Others

### Describe
add nearest interp v2 trt plugin converter, the inference speed of ppyolov2 r50vd dcn before and after this PR can be seen as follows:
f349a7764059f721ad5fd4e48aec0aba

### CODE REVIEW

**Review by qingqing01 — COMMENTED** [review]
please add UT

File: paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc [hunk 1]
```diff
@@ -0,0 +1,116 @@
+/* Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```
Comment (shangzhizhou) [inline paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc]: 2018 -> 2021

File: paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+      int h_axis = (data_layout == framework::DataLayout::kNCHW) + with_dynamic;
+      int w_axis =
+          (data_layout == framework::DataLayout::kNCHW) + 1 + with_dynamic;
+
+      scale_h =
+          static_cast<float>(out_h) / static_cast<float>(in_dim.d[h_axis]);
+      scale_w =
+          static_cast<float>(out_w) / static_cast<float>(in_dim.d[w_axis]);
+    } else {
```
Comment (shangzhizhou) [inline paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc:75]: out_h、out_w、scale的合法性检查，请放到op_teller，不满足约束的，不进trt

File: python/paddle/fluid/tests/unittests/ir/inference/test_trt_nearest_interp_v2_op.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```
Comment (shangzhizhou) [inline python/paddle/fluid/tests/unittests/ir/inference/test_trt_nearest_interp_v2_op.py]: 2020 -> 2021

File: paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc (hunk 1 above)
Comment (wangxinxin08) [reply paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc]: done, thx!

File: python/paddle/fluid/tests/unittests/ir/inference/test_trt_nearest_interp_v2_op.py (hunk 3 above)
Comment (wangxinxin08) [reply python/paddle/fluid/tests/unittests/ir/inference/test_trt_nearest_interp_v2_op.py]: done, thx!

File: paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc (hunk 2 above)
Comment (wangxinxin08) [reply paddle/fluid/inference/tensorrt/convert/nearest_interp_v2_op.cc:75]: done, thx!

**Review by OliverLPH — APPROVED** [review]
LGTM for infer_ut

**Review by XieYunshen — APPROVED** [review]
LGTM for `set_tests_properties(test_trt_nearest_interp_v2_op PROPERTIES TIMEOUT 30)`

**Review by shangzhizhou — APPROVED** [review]
LGTM

**Review by jiweibo — APPROVED** [review]

**Review by heavengate — APPROVED** [review]

**Pull request merged**